src/Components/ArticlePage.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const ArticlePage = () => {
+  const { id } = useParams();
+  console.log(useParams());
+  const [article, setArticle] = useState(null);
```

Comment:
If we expect article to eventually be an object, I would prefer to see this initialised as an empty object, just for consistency and readability